__tests__/pages/components/__snapshots__/Landing.js.snap

```diff
@@ -2,71 +2,77 @@
 
 exports[`Landing <LandingNav /> 1`] = `
 <nav
-  className="navbar navbar-expand-lg navbar-light justify-content-between"
+  className="navbar navbar-expand-lg navbar-light justify-content-between bg-white"
 >
-  <a
-    className="navbar-brand text-primary font-weight-bold"
-    href="#"
-  >
-    C0D3
-  </a>
   <div
-    id="navbarNav"
+    className="container"
   >
+    <a
+      className="navbar-brand text-primary font-weight-bold"
+      href="#"
+    >
+      C0D3
+    </a>
     <div
-      className="navbar-nav collapse navbar-collapse"
+      id="navbarNav"
     >
-      <a
-        className="nav-item nav-link active"
-        href="#"
+      <div
+        className="navbar-nav collapse navbar-collapse"
       >
-        Home 
-        <span
-          className="sr-only"
+        <a
+          className="nav-item nav-link active"
+          href="#"
         >
-          (current)
-        </span>
-      </a>
-      <a
-        className="nav-item nav-link"
-        href="#learning"
-      >
-        Learning Process
-      </a>
-      <a
-        className="nav-item nav-link"
-        href="#journey"
-      >
-        Journey
-      </a>
+          Home 
+          <span
+            className="sr-only"
+          >
+            (current)
+          </span>
+        </a>
+        <a
+          className="nav-item nav-link"
+          href="#learning"
+        >
+          Learning Process
+        </a>
+        <a
+          className="nav-item nav-link"
+          href="#journey"
+        >
+          Journey
+        </a>
+        <a
+          className="nav-item nav-link"
+          href="https://chat.c0d3.com"
```

Comment:
hmmm... `help` should really me an email to `song@garagescript.org` temporarily. In the future, it should go to `contact@c0d3.com` that will get posted into our chatroom for us to respond back. but until then.... `chat.c0d3.com` does not help because they don't have account.